src/Data/Bookmark.elm

```diff
@@ -122,24 +108,14 @@ findByTextileQuery textileQuery =
     findByQuery (Textile textileQuery)
 
 
-scope : Bookmark -> Scope
-scope bookmark =
-    case bookmark.query of
-        Food _ ->
-            Scope.Food
-
-        Textile _ ->
-            Scope.Textile
-
-
 sort : List Bookmark -> List Bookmark
 sort =
     List.sortBy (.created >> Time.posixToMillis) >> List.reverse
 
 
 toId : Bookmark -> String
 toId bookmark =
-    Scope.toString (scope bookmark) ++ ":" ++ bookmark.name
+    bookmark.name
```

Comment:
The idea here was to ensure always having unique ids by prefixing them with the scope, whatever the bookmark name. Imagine using `test` as a food bookmark then `test` as a textile one to understand the issue. I think we should keep addressing it, though I'm gonna leave that decision up to you.